### PR TITLE
Automatically Create Schema

### PR DESCRIPTION
**Pull request by heathharrelson**
Automatically run DDL script to create the COACH schema the first time that the MySQL container starts.

### CODE REVIEW

File: docker-compose.override.yml [hunk 1]
```diff
@@ -15,6 +15,7 @@ services:
     image: mysql:5.7
```
Comment (heathharrelson) [inline docker-compose.override.yml:15]: Should this be MySQL 8, since that's what Matt is using?

Comment (aeyates) [reply docker-compose.override.yml:15]: Yeah, I think so.

File: docker-compose.override.yml [hunk 2]
```diff
@@ -15,6 +15,7 @@ services:
...
     volumes:
       - ./docker_data/mysql:/var/lib/mysql
+      - ./db-schema.sql:/docker-entrypoint-initdb.d/db-schema.sql
```
Comment (aeyates) [inline docker-compose.override.yml:18]: I didn't know about this. So simply copy it into this location in the container and voila?! Very cool.

**Review by aeyates — APPROVED** [review]

File: docker-compose.override.yml (hunk 2 above)
Comment (heathharrelson) [reply docker-compose.override.yml:18]: Yes, it's one of the features of the MySQL image. It's described in [the "Initializing a fresh instance" on the image's Docker Hub page](https://hub.docker.com/_/mysql). The PostgreSQL image has something similar.

**Pull request merged**